setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+from glob import glob
+from os.path import basename
+from os.path import splitext
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+with open("README.md", "r") as fh:
```

Comment:
You'll want to specify what kind of encoding you're using if this is going to run in kanopy. Pip will get confused otherwise.

Something like this if it's actually utf-8.
```
with open("README.md", "r", encoding='utf-8') as fh:
```

Reply:
This shouldn't matter. This code is reading the readme file to create a description for the packaging information. So it is really only important if we are building the package in order to upload it to PyPi. Since we aren't planning to upload this project to PyPi and we wouldn't upload the package from Kanopy if we were, we don't really need to specify the encoding.

Reply:
I'm fairly certain an error will still be thrown when the module is built locally in the docker container. But I took another look and the encoding is being specified in the web_server script anyways so this is probably fine. Going to resolve.